frontend/packages/ceph-storage-plugin/src/components/kms-config/kms-config.tsx

```diff
@@ -83,3 +90,27 @@ export const KMSConfigure: React.FC<KMSConfigureProps> = ({
     </div>
   );
 };
+
+export const createClusterKmsResources = (kms: KMSConfig, provider = ProviderNames.VAULT) => {
```

Comment:
Why did we move it here instead of `utils.tsx` file ? Just confirming...

Reply:
i am importing ```VaultAuthMethodMapping``` from vault-config.tsx and using inside this function. I can't import vault-config inside utils.ts. it will cause circular dependency. 

So I decided to move here.

Reply:
ack...